server/index.js

```diff
@@ -1,30 +1,129 @@
+/* global require */
 const express = require('express');
 const app = express();
-const {getListing} = require('../database/index.js');
+const {getListing, Listing} = require('../database/index.js');
 const cors = require('cors');
-
+const bodyParser = require('body-parser');
 app.use(cors());
 
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: true}));
+// eslint-disable-next-line no-undef
 app.use(express.static(__dirname + '/../client/public/dist'));
 
-app.get('/api/intro/:id', function (req, res) {
-  var id = req.params.id;
+app.get('/api/intro/:listingId', function (req, res) {
+  var id = req.params.listingId;
   getListing(id, function(err, result) {
     if (err) {
-      console.log('fail to get intro')
+      console.log('fail to get intro');
     } else {
-      res.send(result)
+      res.send(result);
     }
   });
 });
 
-app.get('/app.js', cors(), function (req, res) {
-  res.sendFile(path.join(__dirname, '../client/public/dist/app.js'))
+// POST Request - Adds a record to
+app.post('/api/intro/', (req, res) => {
+  /**
+   * Schema for db.listings():
+   * {
+   *    listingNumber: Number,
+   *    photos: array(5 to 6, urlStrings), // 0 - 6.
+   *    title: String,
+   *    description: String
+   * }
+   *
+   */
+
+  Listing.create(req.body)
+    .then((result) => {
+      res.send(result);
+      res.status(200);
+    })
+    .catch((err) => {
+      res.send(err);
+      res.status(500);
+    });
 });
 
-let port = 3002;
-app.listen(port, function() {
-  console.log(`listening on port ${port}`);
+// Delete Request
+/**
+ * @param listingID - listingId used to find documents
+ * Delete listing
+ */
+app.delete('/api/intro/:listingId', (req, res) => {
+  Listing.deleteOne({ listingId: req.params.listingID})
+    .exec()
```

Comment:
The .exec may be not necessary. .then will cause it execute, so you should be able to lose this line.